homeassistant/components/tankerkoenig/sensor.py

```diff
@@ -61,13 +59,13 @@ async def async_update_data():
         _LOGGER,
         name=NAME,
         update_method=async_update_data,
-        update_interval=tankerkoenig.update_interval,
+        update_interval=timedelta(minutes=DEFAULT_SCAN_INTERVAL),
```

Comment:
This might break some imported configurations, since the scan interval cannot be configured anymore. As @frenck mentioned, it is not desired/allowed anymore, so the change itself is OK. But this should probably be mentioned explicitly in the breaking changes part.

Reply:
yeah you are right, will add a note to the breaking change, that the scan interval will be hard set to 30min